skycatalogs/objects/sncosmo_object.py

```diff
@@ -12,6 +12,13 @@ class SncosmoObject(BaseObject):
     def _get_sed(self, mjd=None):
         params = self.get_native_attribute('salt2_params')
         sn = SncosmoModel(params=params)
+        if mjd is None:
```

Comment:
The same comments I made for the `snana_object.py` code regarding where to check if `mjd is None` apply here as well.